### DISCUSSION TITLE
Add CSS variables for `font-weight`

### DISCUSSION THREAD

**Discussion by oemueller** [SchwarzIT/onyx#1100] [Ideas]
Hi, 
I noticed  that currently `font-weight` is declared with fixed values like 600 or 400 in the components. 

Using CSS variables the values could be maintained more easily in one central location like colors. 

Example: `--onyx-font-weight-bold` or `--onyx-font-weight-loud`

same could be done for font-sizes, I bet you already discussed this 🙂

**Comment by larsrickert** [comment]
There are some [pre-defined CSS font weights,](https://developer.mozilla.org/en-US/docs/Web/CSS/@font-face/font-weight#common_weight_name_mapping) maybe we should just use them instead of numbers?

**Reply by oemueller** [reply]
possible, but the idea was to manage all usages of "600/bold" in one place / css variable. With that you can change the value very quickly to e.g. 500 or 700, just like with color values

**Reply by JoCa96** [reply]
@jannick-ux needs to say if that's possible with Figma

**Comment by larsrickert** [comment]
@oemueller There are now new CSS variables for font, including font-weight, see our [docs](https://onyx.schwarz/variables/fonts.html).
They are available since version 1.0.0-beta.248
